### PR TITLE
Updated - starting to reduxify

### PR DESCRIPTION
**Pull request by dbaile6**

### CODE REVIEW

**Comment by dbaile6** [comment]
@janelleany

File: myecommerce/src/Main.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    return (
+      <Router>
+
+    <main>
+      <Switch>
+      <Route path="/" exact component={LandingPage} />
+      <Route path="/cart" exact component={CartSite} />
+      <Route path="/signin" exact component={SignInPage} />
+      <Route path="/products" exact component={ProductSite} />
+      <Route path="/products/:category" exact component={CategoryPage} />
```
Comment (janelleany) [inline myecommerce/src/Main.js:26]: I think having "exact" in line 26 might be ambiguous for the different categories, but I'm not sure.

File: myecommerce/src/db/db.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const Sequelize = require('sequelize');
```
Comment (janelleany) [inline myecommerce/src/db/db.js:1]: This looks super cool. I'll have to look this up.

File: myecommerce/src/CartSite.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
```
Comment (janelleany) [inline myecommerce/src/CartSite.js:2]: all these class components look great and ready to mapStateToProps and mapDispatchToProps and then connect. Awesome

**Review by janelleany — COMMENTED** [review]
this looks great

**Pull request merged**